app/enterprise/0.36-x/plugins/statsd.rules.yaml

```diff
@@ -1,40 +1,4 @@
 mappings:
-# by API
```

Comment:
Do these need corresponding matches for Service?

Reply:
Yes we have those matches for Service below

Reply:
Of course 😆